tasks/79_calculate_primoral_of_a_number/calculate_primoral_of_a_number.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable for-direction */
+//  I modified slightly my code used in previous exercises.
+
+export const numPrimorial = ( num ) => {
+  let primesArray = [];
+  for ( let i = 2; i > 0; i ++) {
+    if ( primesArray.length + 1 <= num ) {
+      getPrime( i, primesArray );
+    } else {
+      break;
+    }
+  }
+  return primesArray.reduce(( prev, next ) => prev * next, 1);
+};
+
+const getPrime = ( num, arr ) => {
+  for ( let i = 2; i <= Math.sqrt(num); i++ ) {
+    if ( num % i === 0 ) {
+      return false;
+    }
+  }
+  return arr.push(num);
```

Comment:
there is no reason for `return` statement as anyway you don`t use length of array later

Reply:
Done